### PR TITLE
Added an example showing how to use Gemini with LangChain.js in Node

### PR DESCRIPTION
**Pull request by pcoet**
## Description of the change

Added an example showing how to use Gemini models with LangChain.js in Node.js.

## Motivation

LangChain is a popular framework for developing LLM apps. This quickstart is a runnable example that illustrates some of the functionality discussed in the existing LangChain.js docs:

* [LangChain.js: Google](https://js.langchain.com/docs/integrations/platforms/google)
* [LangChain.js: ChatGoogleGenerativeAI](https://js.langchain.com/docs/integrations/chat/google_generativeai)

Note: Eventually I'd like to add a TypeScript version of this example, and perhaps also a web version. I might also add a small server implementation to make it a more typical Node app.

## Type of change
Choose one: (Bug fix | Feature request | Documentation | Other): Other (example)

## Checklist

- [x] I have performed a self-review of my code.
- [x] I have added detailed comments to my code where applicable.
- [x] I have verified that my change does not break existing code.
- [x] My PR is based on the latest changes of the main branch (if unsure, please run `git pull --rebase upstream main`).
- [x] I am familiar with the [Google Style Guide](https://google.github.io/styleguide/) for the language I have coded in.
- [x] I have read through the [Contributing Guide](https://github.com/google/generative-ai-docs/blob/main/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://cla.developers.google.com/about).

### CODE REVIEW

File: examples/gemini/javascript/langchain_quickstart_node/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Run
+
+```
+npm start
+```
+
+## Learn more
+
+Google AI also provides a
+[JavaScript SDK](https://github.com/google/generative-ai-js) for interacting
```
Comment (8bitmp3) [inline examples/gemini/javascript/langchain_quickstart_node/README.md:47]: Nit: Maybe "the ... SDK" (officially "Google AI JavaScript SDK")

**Review by 8bitmp3 — APPROVED** [review]

File: examples/gemini/javascript/langchain_quickstart_node/main.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      'human',
+      'What is the meaning of life?',
+    ],
+  ]);
+
+  console.log(result);
+}
+
+/**
+ * Creates a Gemini Pro Vision multimodal chat model, invokes the model with an
```
Comment (markmcd) [inline examples/gemini/javascript/langchain_quickstart_node/main.js:42]: This is a little confusing (not your comment, just the structure of things) - we don't actually expose a multimodal chat model, but I think LC is wrapping up this "Chat" service using the straight text-completion service. The class is called "ChatGoogleGenerativeAI" so I think it's fine, just potentially confusing.

**Review by markmcd — APPROVED** [review]
I added an `npm i` step, and updated `package-lock` to match the package.

**Comment by markmcd** [comment]
FYI: I'm adding this to the site (/examples) in cl/598689574.

**Comment by pcoet** [comment]
@markmcd Thanks for adding the `npm install`!

**Pull request merged**